kivy/input/providers/mtdev.py

```diff
@@ -220,7 +220,8 @@ def process(points):
                     queue.append((action, touch))
 
             def normalize(value, vmin, vmax):
-                return (value - vmin) / float(vmax - vmin)
+                divisor = float(vmax - vmin) if vmax > vmin else 1
```

Comment:
As it hasn't returned absolute values before explicitly, I believe this is wrong (what about the negative values?). I'm not sure what are those values, but something like `... if vmax != vmin else 1` should be good at the first look, however if there is a possibility at least one of the values is a float, I'd rather like to see more hm, float-friendly handling i.e. let the error appear, but handle it directly:

    try:
        return (value - vmin) / float(vmax - vmin)
    except ZeroDivisionError:  # it's both in py2 and py3
        return (value - vmin)

Reply:
Good point, I modified code.